jobs/aws/eks-anywhere-build-tooling/build-tooling-attribution-files-periodics-release-0.6.yaml

```
@@ -0,0 +1,95 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+periodics:
+  - name: build-tooling-attribution-files-periodic
```

```suggestion
  - name: build-tooling-attribution-files-periodic-release-0-6
```
or something that follows the regex `[A-Za-z0-9-._]+`